imagestreams/imagestreams.yaml

```diff
@@ -15,10 +15,10 @@
         app_versions: ["10.3", "10.5"]
 
       - name: CentOS Stream 9
-        app_versions: ["10.5"]
+        app_versions: ["10.5", "10.11"]
```

Comment:
Don't we want to set 10.11-el9 as latest for CentOS Stream too?

Reply:
Why not?

Reply:
Yes, I would expect having both c9s and rhel9 share the same latest image version.